website/content/docs/services/discovery/dns-forwarding/enable.mdx

```diff
@@ -0,0 +1,442 @@
+  dump-file       "/var/named/data/cache_dump.db";
+  statistics-file "/var/named/data/named_stats.txt";
+  memstatistics-file "/var/named/data/named_mem_stats.txt";
+  allow-query     { localhost; };
+  recursion yes;
+
+  dnssec-enable no;
+  dnssec-validation no;
+
+  /* Path to ISC DLV key */
+  bindkeys-file "/etc/named.iscdlv.key";
+
+  managed-keys-directory "/var/named/dynamic";
+};
+
+include "/etc/named/consul.conf";
+```
+
+</CodeBlockConfig>
+
+### Zone file
+
+Set up a zone for your Consul managed records in `consul.conf`.
+
+<CodeBlockConfig filename="/etc/named/consul.conf">
+
+```dns-zone-file
+zone "consul" IN {
+  type forward;
+  forward only;
+  forwarders { 127.0.0.1 port 8600; };
+};
+```
+
+</CodeBlockConfig>
+
+## Unbound
+
+[Unbound](https://www.unbound.net/) is a validating, recursive, caching DNS resolver. It is designed to be fast and lean and incorporates modern features based on open standards.
+
+<Note>
+
+ If your distribution uses systemd, you may need to disable `systemd-resolved` prior to following this example. 
+ 
+</Note>
+
+Unbound is typically configured in a `unbound.conf` file or in a series of files in the `/etc/unbound/unbound.conf.d` directory. 
+
+In this example you will use a `consul.conf` file in the `/etc/unbound/unbound.conf.d` directory.
+
+Add the following settings to your Unbound configuration file:
+
+<CodeBlockConfig filename="/etc/unbound/unbound.conf.d/consul.conf">
+
+```plaintext
+#Allow insecure queries to local resolvers
+server:
+  do-not-query-localhost: no
+  domain-insecure: "consul"
+
+#Add consul as a stub-zone
+stub-zone:
+  name: "consul"
+  stub-addr: 127.0.0.1@8600
+```
+
+</CodeBlockConfig>
+
+You may have to add the following line to the bottom of your `/etc/unbound/unbound.conf` file for the new configuration to be included.
+
+<CodeBlockConfig filename="/etc/unbound/unbound.conf">
+
+```plaintext
+include: "/etc/unbound/unbound.conf.d/*.conf"
+```
+
+</CodeBlockConfig>
+
+## iptables
+
+[iptables](https://www.netfilter.org/projects/iptables/index.html) is a generic firewalling software, part of the netfilter project, that allows you to define traffic rules for your system.
+
+In situations where you don't have a local DNS server on the Consul agent node you can use `iptables` to forward DNS requests on port `53` to the Consul agent running on the same machine without using a secondary service.
+
+This configuration realizes a full DNS forwarding, meaning that all DNS queries for the host will be forwarded to Consul and not only the ones for the `.consul` *Top Level Domain* (TLD). Since Consul by default only resolves the `.consul` TLD you ***must*** set the [`recursors`](/consul/docs/agent/config/config-files#recursors) flag if you want your node to be able to resolve other domains when using `iptables` configuration.
+
+<Note>
+
+In case you use DNS relay hosts in your network, they _should not_ be on the same host, otherwise the redirects will intercept the traffic.
```

Comment:
```suggestion
In case you use DNS relay hosts in your network, they should not be on the same host, otherwise the redirects intercept the traffic.
```